common/flatpak-dir.c

```diff
@@ -11794,15 +11832,45 @@ flatpak_dir_list_remote_config_keys (FlatpakDir *self,
   return NULL;
 }
 
-char **
+static void
+add_subdirs (GPtrArray *res,
+             GFile     *parent)
+{
+  g_autoptr(GFileEnumerator) dir_enum = NULL;
+
+  dir_enum = g_file_enumerate_children (parent,
+                                        G_FILE_ATTRIBUTE_STANDARD_NAME ","
+                                        G_FILE_ATTRIBUTE_STANDARD_TYPE,
+                                        G_FILE_QUERY_INFO_NONE,
+                                        NULL, NULL);
+  if (dir_enum == NULL)
+    return;
+
+  while (TRUE)
+    {
+      GFileInfo *info;
+      GFile *path;
+
+      if (!g_file_enumerator_iterate (dir_enum, &info, &path, NULL, NULL) ||
+          info == NULL)
+        break;
+
+      if (g_file_info_get_file_type (info) == G_FILE_TYPE_DIRECTORY)
+        g_ptr_array_add (res, g_object_ref (path));
+    }
+}
+
+GPtrArray *
 flatpak_dir_get_sideload_repo_paths (FlatpakDir *self)
 {
-  GKeyFile *config = flatpak_dir_get_repo_config (self);
+  g_autoptr(GFile) sideload_repos_dir = flatpak_dir_get_sideload_repos_dir (self);
+  g_autoptr(GFile) sideload_runtime_repos_dir = flatpak_dir_get_runtime_sideload_repos_dir (self);
```

Comment:
Would be nice if the variable name matched the function, `runtime_sideload_repos_dir`